src/ResourceManager/Profile/Commands.Profile/Account/AddAzureRmAccount.cs

```diff
@@ -119,12 +120,12 @@ protected override void ProcessRecord()
                 azureAccount.SetProperty(AzureAccount.Property.Tenants, new[] { Tenant });
             }
 
-            if( AzureRMCmdlet.DefaultProfile == null)
+            if( AzureRmProfileProvider.Instance.Profile == null)
             {
-                AzureRMCmdlet.DefaultProfile = new AzureRMProfile();
+                AzureRmProfileProvider.Instance.Profile = new AzureRMProfile();
             }
 
-            var profileClient = new RMProfileClient(AzureRMCmdlet.DefaultProfile);
+            var profileClient = new RMProfileClient(AzureRmProfileProvider.Instance.Profile);
```

Comment:
Any reference to `AzureRmProfileProvider.Instance` can be replaced with `DefaultProfile`